velox/vector/tests/VectorMaker.h

```diff
@@ -193,6 +193,22 @@ class VectorMaker {
     return flatVector;
   }
 
+  /// Create a DECIMAL FlatVector from native DECIMAL values, precision, scale.
+  ///
+  /// Elements are nullable.
+  ///
+  /// Examples:
+  ///  auto flatVector = decimalFlatVectorNullable({1, std::nullopt, 3}, 8, 1);
+  FlatVectorPtr<ShortDecimal> decimalFlatVectorNullable(
```

Comment:
Would it be helpful to provide a non-nullable version as well to allow for shorter code when nulls are not used?